packages/data-context/src/actions/DataEmitterActions.ts

```diff
@@ -109,7 +123,7 @@ export class DataEmitterActions extends DataEmitterEvents {
         if (!hasSentInitial && sendInitial) {
           hasSentInitial = true
 
-          return { done: false, value: {} }
+          return { done: false, value: undefined }
```

Comment:
+1 here. I think I noticed this at one point, but forgot to ask about it/change it.  It was weird to me that we did {} in this scenario, but undefined in others